src/herder/TxSetFrame.cpp

```diff
@@ -237,13 +248,13 @@ TxSetFrame::surgePricingFilter(Application& app)
             surgeQueue.pop();
             // inspect the top candidate queue
             auto& curTopTx = cur->front();
-            if (curTopTx->getOperations().size() <= opsLeft)
+            size_t opsCount = curTopTx->getEnvelope().tx.operations.size();
```

Comment:
Should this be `size_t opsCount = maxIsOps ? curTopTx->getEnvelope().tx.operations.size() : MAX_OPS_PER_TX`?

Reply:
it ends up being equivalent as opsLeft is a multiple of `MAX_OPS_PER_TX`, but I agree the code is easier to understand by just computing the adjusted `opsCount` right away